packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePicker.test.tsx

```diff
@@ -97,4 +98,10 @@ describe('PeoplePicker', () => {
 
     ReactDOM.unmountComponentAtNode(root);
   });
+
+  isConformant({
+    Component: NormalPeoplePicker,
+    displayName: 'NormalPeoplePicker',
+    disabledTests: ['has-top-level-file', 'is-static-property-of-parent', 'name-matches-filename'],
```

Comment:
Probably `is-static-property-of-parent` should be universally disabled for OUFR since we don't use that convention.